cmd/timescale-prometheus/main.go

```diff
@@ -47,6 +47,7 @@ type config struct {
 }
 
 const (
+	version           = "0.1.0-dev"
```

Comment:
Maybe we need to make this a placeholder string so it can be replaced when releasing by an arg, or when the CI is building

Reply:
Maybe we could include the commit? Using `go generate` since we are already using that.

Reply:
I like the idea of a git commit a lot